faq.md

```diff
@@ -64,13 +64,13 @@ or stickers :P).
 Considering that a few contributors are investing a large amount of time in the
 project, we are thinking about ways to make the project sustainable.
 
-This could be achieved via donations, public fundings, and a few contributors
+This could be achieved via donations, crowdfunding, and a few contributors
```

Comment:
Crowdfunding is different from what I think about public funding, which should maybe called grants ?